### PR TITLE
Fix naming convention for SUBSCRIPTION_EVENT_NAMES & ORDER_STATE

### PR DESCRIPTION
**Pull request by adrake33**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -156,11 +156,11 @@ interface BlockSubscriptionCallbacks {
   onRemoved?: (block: Block) => void;
 }
 
-export enum OrderState {
-  All = "ALL",
-  Cancelled = "CANCELLED",
```
Comment (epheph) [inline index.d.ts]: This will be a larger if this drops the "L". if we want to do it, we should line it up with another DB forced resync

Comment (adrake33) [reply index.d.ts]: Oh, but isn't AN using "OrderState.Canceled", not "OrderState.Cancelled"?

Comment (pgebheim) [reply index.d.ts]: We can, and should, do a migration to fix the DB instead of doing force re-syncs.

**Comment by adrake33** [comment]
Discussed this further with @epheph and @pg, and we decided no further changes need to be made in order to use OrderState.CANCELED in augur-node, because this is already the format being used there.

**Pull request merged**